### PR TITLE
Implemented Base api endpoints

### PR DESCRIPTION
**Pull request by Alekaei**
Added base api endpoints to build on to

### CODE REVIEW

File: EejitCMS.API/Controllers/AccountController.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+		// GET api/account/:id
+		[HttpGet("{id:string}")]
+		public Account Get(string id)
+		{
+			throw new NotImplementedException ();
+		}
+
+		// PUT api/account/:id
+		[HttpPost("{id:string}")]
```
Comment (ocanty) [inline EejitCMS.API/Controllers/AccountController.cs]: HttpPost on PUT?

File: EejitCMS.API/Models/Account.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EejitCMS.API.Models
+{
+	public class Account
+	{
+		public int Id { get; set; }
```
Comment (ocanty) [inline EejitCMS.API/Models/Account.cs]: consider using strings for ids

makes the number immutable, + if we run multiple backend servers for load balancing it doesn't mess stuff up if we use sequential identifiers
https://softwareengineering.stackexchange.com/a/361405

File: EejitCMS.API/Models/Media.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EejitCMS.API.Models
+{
+	public class Media
+	{
+		public int Id { get; set; }
```
Comment (ocanty) [inline EejitCMS.API/Models/Media.cs]: see above

File: EejitCMS.API/Models/Page.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EejitCMS.API.Models
+{
+	public class Page
+	{
+		public int Id { get; set; }
```
Comment (ocanty) [inline EejitCMS.API/Models/Page.cs]: .

File: EejitCMS.API/Models/Account.cs (hunk 2 above)
Comment (ericm) [reply EejitCMS.API/Models/Account.cs]: Agree with @ocanty on making them immutable but keep them as ints.

File: EejitCMS.API/Controllers/AccountController.cs (hunk 1 above)
Comment (ericm) [reply EejitCMS.API/Controllers/AccountController.cs]: Just change the comment `// PUT api/account/:id`.

**Review by ericm — COMMENTED** [review]
All good but consider making the ids immutable. That can be implemented later though.

**Review by ericm — APPROVED** [review]

**Pull request merged**